mlrun/utils/notifications/notification/base.py

```diff
@@ -59,6 +69,13 @@ def load_notification(
     ) -> None:
         self.params = params or {}
 
+    @classmethod
```

Comment:
why is it a classmethod and not a self?

Reply:
The answer for the both question is that I call it on [validate_notification_params](https://github.com/roei3000b/mlrun/blob/9ad20791019778b0322c929fc41c21fd933fc4a9/mlrun/model.py#L760)

So:

1. It should be class method because on [this line](https://github.com/roei3000b/mlrun/blob/9e0ca0db3d8db1cf50e6af16c988454d51c5d69d/mlrun/model.py#L756) we use the class object for enrich_default_params.
2. It should be public because we call it here.